source/open_node/ast_visitor/ast_visitor.py

```diff
@@ -0,0 +1,74 @@
+from typing import Optional, TypeVar, Generic
+
+from open_node.ast import OpenNode
```

Comment:
We should consider changing the package name to `openqasm` and then securing that name on PyPi.

Reply:
+1

Reply:
Unfortunately, openqasm was [claimed](https://pypi.org/project/openqasm/) 2 years ago. openqasm_parser and openqasm_ast are still available. Any preference?

Reply:
openqasm_ast +1

Reply:
(My little mentorship team wants to leverage your work in our QIskit Advocates Mentorship Program project and write a code generator from your AST)

Reply:
Changed the package name to openqasm for now.